### ISSUE TITLE
Ensure odd numbers of voting leaders to ensure tie decisions are break

### ISSUE THREAD

**Issue by remyleone** [kubernetes/community#4615]
Could we make an explicit recommendation to have an odd number of voting leaders in all SIG? This could ensure that in case of a debate there is always a tie break with a decision to move things forward.

Already enforced tacitly by @zacharysarah but it could be made more explicit and systematic to have it as a general recommendation.

**Comment by zacharysarah** [comment]
It’s not a pressing issue in any SIG as far as I know, but the time to fix for good governance isn’t in the middle of needing it. +1

**Comment by lavalamp** [comment]
My hot take: I disagree with this. We should not be voting on code. I can't think of any situation in API Machinery where a 3rd co-TL or co-chair would have changed the outcome, and I wouldn't want to participate in a forum with the implied amount of conflict needed to make that untrue.

**Comment by bgrant0607** [comment]
Code decisions are based on subproject OWNERS files, not SIG-level votes.

Ref #4617

**Comment by nikhita** [comment]
We discussed this issue in the Private Steering Committee meeting on 16 March 2020. It might be helpful to have a nuanced discussion around this to figure out how to better document our current processes. I've talked to @remyleone offline and we have decided to discuss more about this issue in the next Public Steering Committee meeting on 6 April 2020. I'll send out an email to the steering mailing list before the meeting as a reminder.
